rclcpp/include/rclcpp/memory_strategies.hpp

```diff
@@ -15,16 +15,24 @@
 #ifndef RCLCPP_RCLCPP_MEMORY_STRATEGIES_HPP_
 #define RCLCPP_RCLCPP_MEMORY_STRATEGIES_HPP_
 
-#include <rclcpp/strategies/heap_pool_memory_strategy.hpp>
-#include <rclcpp/strategies/stack_pool_memory_strategy.hpp>
+//#include <rclcpp/strategies/heap_pool_memory_strategy.hpp>
+//#include <rclcpp/strategies/stack_pool_memory_strategy.hpp>
+#include <rclcpp/memory_strategy.hpp>
+#include <rclcpp/strategies/allocator_memory_strategy.hpp>
 
 namespace rclcpp
 {
 namespace memory_strategies
 {
 
-using rclcpp::memory_strategies::heap_pool_memory_strategy::HeapPoolMemoryStrategy;
-using rclcpp::memory_strategies::stack_pool_memory_strategy::StackPoolMemoryStrategy;
+using rclcpp::memory_strategies::allocator_memory_strategy::AllocatorMemoryStrategy;
+//using rclcpp::memory_strategies::heap_pool_memory_strategy::HeapPoolMemoryStrategy;
+//using rclcpp::memory_strategies::stack_pool_memory_strategy::StackPoolMemoryStrategy;
```

Comment:
It looks like you later delete the files, so you might as well remove the `using` and `#include` statements.